lib/guardian.ex

```diff
@@ -366,9 +366,10 @@ defmodule Guardian do
 
       @spec config() :: Keyword.t()
       def config,
-        do: unquote(otp_app)
-            |> Application.get_env(__MODULE__, [])
-            |> Keyword.merge(unquote(opts))
+        do:
```

Comment:
This should probably just be changed to use the block syntax to avoid this change.